pajbot/modules/moderators_refresh.py

```diff
@@ -34,21 +35,19 @@ def update_moderators_cmd(self, bot, source, **rest):
         bot.whisper(source, "Reloading list of moderators...")
         bot.action_queue.submit(self._update_moderators)
 
-    def _update_moderators(self):
+    @time_method
+    def _update_moderators(self) -> None:
         if self.bot is None:
             log.error("_update_moderators failed in ModeratorsRefreshModule because bot is None")
             return
 
         try:
-            moderator_ids = self.bot.twitch_helix_api.fetch_all_moderators(
+            moderators = self.bot.twitch_helix_api.fetch_all_moderators(
                 self.bot.streamer.id, self.bot.streamer_access_token_manager
             )
+            # As per the Helix docs, the broadcaster will not be in the response.
+            moderators.append(UserBasics(self.bot.streamer.id, self.bot.streamer.login, self.bot.streamer.name))
```

Comment:
```suggestion
            moderators.append(self.bot.streamer)
```

:)

https://github.com/pajbot/pajbot/blob/0ef6bd6a5598822ccca459261a4a2a8e9e9c1854/pajbot/bot.py#L147